<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1500</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1500</width>
    <height>477</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>477</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>抽卡期望模拟器 - Numba GUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/res/icon.png</normaloff>:/res/res/icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { color: white; }
QLabel { color: white; }
QPushButton { color: white; }
QCheckBox { color: white; }
QLineEdit, QTextEdit, QSpinBox { background-color: rgba(0,0,0,0); color: white; border: 1px solid rgba(255,255,255,0.6); }
QTextEdit { selection-background-color: rgba(255,255,255,0.2); }
QSpinBox::up-button, QSpinBox::down-button { background: transparent; border: none; }
QSpinBox::up-arrow { image: url(:/res/res/arrow_up.svg); width: 12px; height: 8px; }
QSpinBox::down-arrow { image: url(:/res/res/arrow_down.svg); width: 12px; height: 8px; }
</string>
  </property>
  <widget class="QListView" name="background_list">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1501</width>
     <height>491</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; border: none; outline: none; background-image: url(:/res/res/background.png)</string>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="sims">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>290</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>10000000</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_total5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>105</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当期5星总数</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="want4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="status">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>121</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>就绪</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="want5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_seed">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>290</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>随机种子</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sims">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>76</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>模拟次数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="run_btn">
   <property name="geometry">
    <rect>
     <x>1220</x>
     <y>180</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none; background: transparent; color: black;</string>
   </property>
   <property name="text">
    <string>开始模拟</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="normal">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>177</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>常驻池(含50保底)</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_want4">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>230</y>
     <width>105</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>想要4星数量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_want5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>230</y>
     <width>105</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>想要5星数量</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="total5">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>230</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>901</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none; background: transparent;</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="seed">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>290</y>
     <width>108</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>123456789</string>
   </property>
  </widget>
  <widget class="QPushButton" name="random_seed_btn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>290</y>
     <width>38</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>腾祥嘉丽大圆简</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent; border: 1px solid rgba(255,255,255,0.6); color: white;</string>
   </property>
   <property name="text">
    <string> 随机 </string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
